config/index.js

```diff
@@ -200,7 +200,7 @@ function getDomainConfig(domain, ...configKeys) {
     }
 
     const loadedDomainConfig = domainConfigs[domain];
-    if(loadedDomainConfig) {
+    if(typeof loadedDomainConfig !== 'undefined') {
```

Comment:
Prevent trying to load a configuration file for a domain that doesn't exist. Non-existing domains are given `null` in the configuration object